homeassistant/components/proxmoxve/__init__.py

```diff
@@ -63,22 +60,6 @@
                         vol.Optional(
                             CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL
                         ): cv.boolean,
-                        vol.Required(CONF_NODES): vol.All(
```

Comment:
We need to deprecate this option and mark it as a breaking change.

Removing it as done now, would cause configurations to validate as invalid.

Reply:
Ah I was under the impression it wouldn't with: https://github.com/home-assistant/core/blob/b3e3b1de59dea780d0ac789d96851316933b62ab/homeassistant/components/proxmoxve/__init__.py#L68

How do you handle deprecations normally @frenck ? Would it be fine to mark this as optional but keep the schema ? It would be ignored with this change